front-end/src/components/player-page/stats/performance-overview.tsx

```diff
@@ -53,21 +53,17 @@ function getStatBox(index: number, statGenerator: BaseStatGenerator, performance
 
     const average = sorted.reduce((total, next) => total + statGenerator.getStatValue(next), 0) / sorted.length;
 
-    let rank = 0;
-    let isPlayerInLeaderBoard = false;
-    for(; rank < sorted.length; rank++) {
-        if(sorted[rank].playerPuuid === performanceOverviewProps.playerPuuid) {
-            isPlayerInLeaderBoard = true;
-            break;
-        }
-    }
+    let rank = statGenerator.getPositionInLeaderboard(
+        +statGenerator.getStatString(performanceOverviewProps.playerStats),
+        sorted
+    );
 
     return <StatBox key={index}
                     statToolTip={statGenerator.getToolTip()}
                     statValue={statGenerator.getStatString(performanceOverviewProps.playerStats)}
                     statTitle={statGenerator.getStatTitle()}
                     haveStatsLoaded={statGenerator.canLoadData(performanceOverviewProps.playerStats)}
-                    shouldShowLeaderboard={isPlayerInLeaderBoard && statGenerator.canLoadData(performanceOverviewProps.playerStats)}
+                    shouldShowLeaderboard={sorted.length > 0 && statGenerator.canLoadData(performanceOverviewProps.playerStats)}
```

Comment:
Whats the reason for this change?  If a LB exists it doesn't mean that the player is in it for sure.

Reply:
Yeah but they don't need to be. They can just see where they fall in it

Reply:
That doesnt make sense, we shouldnt be comparing players inter league. a gold player playing in rampage wouldnt have the same stats if they played in dominate.

Reply:
Ok, w.e. It won't make a difference anyway. Reverted.